styles/widgets/material/accordion.material.less

```diff
@@ -16,6 +16,7 @@
 
 
 .dx-accordion {
+    background-color: @accordion-background-color;
```

Comment:
It will be looked strange in material theme. And has the different appearance with a generic theme. Is it what you want?